scripts/setup-app.sh

```diff
@@ -48,7 +48,7 @@ cp -r templates/mobile deploys/mobile
 cd deploys/mobile || exit 1
 
 printf "%s" "$THEME" >theme.json
-PRIMARY=$(grep "primary" theme.json | sed -E "s/.*(#[[:alnum:]]+).*/\1/")
+PRIMARY=$(grep "primary" theme.json | sed -E "s/.*\"primary\":\s?\"(#[[:alnum:]]+).*/\1/")
```

Comment:
```suggestion
PRIMARY=$(grep "primary" theme.json | sed -E "s/.*\"primary\":[[:space:]]?\"(#[[:alnum:]]+).*/\1/")
```

Nice work! I think we may to use the Mac notation for spaces here. Sometimes it doesn't work with the normal symbols. Verify this though.